langchain/src/llms/bedrock.ts

```diff
@@ -50,9 +61,9 @@ class BedrockLLMInputOutputAdapter {
     if (provider === "anthropic") {
       return responseBody.completion;
     } else if (provider === "ai21") {
-      return responseBody.completions[0].data.text;
+      return responseBody.data.text;
```

Comment:
This was wrong before?

Reply:
This was correct before for the non-streaming bedrock api.  But when I change to the bedrock `invoke-with-response-stream` endpoint, these change to what's in this PR currently.